Odata-docs/client/using-poco-classes.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Using POCO classes"
+description: "This tutorial describes how to use POCO classes on the OData Client."
```

Comment:
```suggestion
description: "This tutorial describes how to use POCO classes in the OData Client."
```

Reply:
> 

in/with can be used interchangeably here depending on the context

Reply:
#Resolved